Renderer.js

```diff
@@ -0,0 +1,55 @@
+class Renderer {
+    constructor() {
+    }
+
+    renderUserInfo() {
+        const source = $('#user-template').html();
+        const template = Handlebars.compile(source)
+        $(".user-container").empty();
+        const newHTML = template(this.data.userInfo);
```

Comment:
This function should receive the userInfo as an argument and not access the global class property